meta-luneos/recipes-support/pidgin/pidgin-sipe_1.21.1.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "Protocol plugin for Office 365/Lync/OCS for Adium, Pidgin, Miranda and Telepathy IM Framework"
+SECTION = "webos/services"
+LICENSE = "GPLv2.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+
+DEPENDS = "pidgin intltool-native"
+
+inherit pkgconfig
+inherit autotools
+
+SRC_URI = "${SOURCEFORGE_MIRROR}/sipe/pidgin-sipe-${PV}.tar.xz"
+SRC_URI[md5sum] = "96ea396adb5c8a1bb1749d778892605c"
+SRC_URI[sha256sum] = "651af55d65cbdf5bdbee0366bd5151bec4152c910743aa432204d015893fe444"
+
+PACKAGECONFIG ??= "nss krb5"
+PACKAGECONFIG[nss] = "--enable-nss=yes,--enable-nss=no,nss"
+PACKAGECONFIG[openssl] = "--enable-openssl=yes,--enable-openssl=no,openssl"
+PACKAGECONFIG[krb5] = "--with-krb5=yes,--with-krb5=no,krb5"
+#PACKAGECONFIG[voice_and_video] = "--with-vv=yes,--with-vv=no,libnice gstreamer"
+PACKAGECONFIG[telepathy] = "--enable-telepathy=yes,--enable-telepathy=no,telepathy-glib gmime"
+#PACKAGECONFIG[gssapi_only] = "--enable-gssapi-only=yes,--enable-gssapi-only=no,krb5"
+PACKAGECONFIG[debug] = "--enable-debug=yes,--enable-debug=no,valgrind"
+
+FILES_${PN} += " \
+    ${libdir} \
+    ${datadir}/appdata \
+    ${datadir}/pixmaps \
+"
```

Comment:
I get the following when I don't provide a FILES_${PN}:

WARNING: pidgin-sipe-1.21.1-r0 do_package: QA Issue: pidgin-sipe: Files/directories were installed but not shipped in any package:
  /usr/share/appdata
  /usr/share/appdata/pidgin-sipe.metainfo.xml
  /usr/lib/purple-2/libsipe.so
  /usr/lib/purple-2/libsipe.la
Please set FILES such that these items are packaged. Alternatively if they are unneeded, avoid installing them or delete them within do_install.
pidgin-sipe: 4 installed and not shipped files. [installed-vs-shipped]